build.js

```diff
@@ -680,6 +680,7 @@ var SPLASH_FILES = [
   'portrait1024',
   'portrait1136',
   'portrait2048',
+  'portrait2960',
```

Comment:
We also specify in the manifest @rampr I won't add if manifest does not have the resolution